packages/functional-tests/tests/misc/keyStretchingV2.spec.ts

```diff
@@ -31,7 +33,7 @@ test.describe('key-stretching-v2', () => {
    * There are differences between how react and backbone use authClient and gql-api
    * for these types of operations.
    */
-  ['react', 'backbone'].forEach((mode) => {
+  ['react'].forEach((mode) => {
```

Comment:
I've removed the backbone tests since it can't be reached in the full production rollut.